### DISCUSSION TITLE
exportArchive: "Runner.app" requires a provisioning profile with Apple features.

### DISCUSSION THREAD

**Discussion by flant-team-zulu** [codemagic-ci-cd/codemagic-docs#2378] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I was generate cert for Apple Distribution from API 
also i did provisioner profiles (app store) for bundle id with capabilities(and fetch it to codemagic):
Apple Pay Payment Processing, Associated Domains, iCloud, In-App Purchase, Push Notifications, Sign In with Apple
   
Use this codemagic.yaml:
 environment:
      ios_signing:
        distribution_type: app_store
        bundle_identifier: xxxxxxx
      vars:
        APP_ID: xxxxxx
        ENV: dev
    scripts:
      - name: Build app
        script: |
          xcode-project use-profiles
          flutter clean
          flutter pub get
          flutter pub run build_runner build --delete-conflicting-outputs
          flutter build ipa --release --flavor=${ENV}app --dart-define=ENV=$ENV -v
          mkdir -p "build/ios/ipa_$ENV"
          cp build/ios/ipa/*.ipa "build/ios/ipa_$ENV"
    artifacts:
      - build/ios/ipa_$ENV/*.ipa

### Expected results

*.ipa files from build

### Actual results

[   +3 ms] Building App Store IPA... (completed in 1,943ms)
[   +1 ms] Encountered error while creating the IPA:
[   +2 ms] error: exportArchive: "Runner.app" requires a provisioning profile with the Apple Pay, Associated Domains, Push Notifications, and Sign in with Apple features.

### Build id (optional)

65ce089b0d5e23751425b42f

**Comment by himesh-cm** [comment]
Hi @flant-team-zulu,
It looks like you were able to figure it out by adding the missing `export-options-plist` flag. Regarding missing capablities you can enable them in your workflow using Codemagic CLI tools, you can find more information about it [here](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/bundle-ids/enable-capabilities.md)

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
